app/model/SupportPhoneDetails.scala

```diff
@@ -0,0 +1,11 @@
+package model
```

Comment:
I would start moving this kind of code in the `view` package (perhaps in `view.model`). This helps conveying the fact that it is only a presentation concern.